### PR TITLE
Pid 567 send notification when mpt credentials are published

### PR DESCRIPTION
**Pull request by martinsaporiti**
* This PR has the code for sending push notifications when the state is published. :rocket:
* Push notifications are sent only when the credential (the link indeed) has mtp proof as not null :helicopter:

### CODE REVIEW

**Review by javip97 — APPROVED** [review]
LGTM

File: internal/repositories/claims.go [hunk 1]
```diff
@@ -876,6 +876,67 @@ func (c *claims) GetClaimsIssuedForUser(ctx context.Context, conn db.Querier, id
...
+		FROM claims
+		WHERE claims.identifier = $1
+		AND identity_state = $2
+		AND mtp_proof IS NOT NULL
+	`
+	rows, err := conn.Query(ctx, query, did.String(), state)
+	if err != nil {
+		return nil, err
+	}
+
```
Comment (x1m3) [inline internal/repositories/claims.go:907]: defer rows.Close()

**Review by x1m3 — CHANGES_REQUESTED** [review]

File: internal/repositories/claims.go [hunk 2]
```diff
@@ -876,6 +876,67 @@ func (c *claims) GetClaimsIssuedForUser(ctx context.Context, conn db.Querier, id
...
+			&claim.CredentialStatus,
+			&claim.Revoked,
+			&claim.CoreClaim)
+		if err != nil {
+			return nil, err
+		}
+		claims = append(claims, &claim)
+	}
+
+	defer rows.Close()
```
Comment (x1m3) [inline internal/repositories/claims.go:936]: This should go up

**Review by x1m3 — CHANGES_REQUESTED** [review]
There is a defer rows.Close that should go up in the code

**Review by x1m3 — APPROVED** [review]
Nice.

**Pull request merged**